### ISSUE TITLE
build/validation issue

### ISSUE THREAD

**Issue by mro** [Megalithicirelandmap/Megalithicirelandmap.github.io#4]
preparation:

```
$ xmllint --version
xmllint: using libxml version 20902
   compiled with: Threads Tree Output Push Reader Patterns Writer SAXv1 FTP HTTP DTDValid HTML Legacy C14N Catalog XPath XPointer XInclude ISO8859X Unicode Regexps Automata Expr Schemas Schematron Modules Debug Zlib 
$ git clone https://github.com/Megalithicirelandmap/Megalithicirelandmap.github.io.git
$ cd Megalithicirelandmap.github.io/
```

expected:

```
$ sh tools/gpxlint --noout monuments.gpx
xmllint --relaxng /Users/mro/Documents/Megalithicirelandmap/tools/monuments.rng --noout monuments.gpx
monuments.gpx validates
```

actual:

```
$ sh tools/gpxlint --noout monuments.gpx
monuments.gpx:518: element cmt: Relax-NG validity error : Error validating datatype string
monuments.gpx:518: element cmt: Relax-NG validity error : Element cmt failed to validate content
monuments.gpx:614: element cmt: Relax-NG validity error : Error validating datatype string
monuments.gpx:614: element cmt: Relax-NG validity error : Element cmt failed to validate content
monuments.gpx:878: element cmt: Relax-NG validity error : Error validating datatype string
monuments.gpx:878: element cmt: Relax-NG validity error : Element cmt failed to validate content
monuments.gpx:920: element cmt: Relax-NG validity error : Error validating datatype string
monuments.gpx:920: element cmt: Relax-NG validity error : Element cmt failed to validate content
monuments.gpx:1004: element cmt: Relax-NG validity error : Error validating datatype string
monuments.gpx:1004: element cmt: Relax-NG validity error : Element cmt failed to validate content
monuments.gpx:1010: element cmt: Relax-NG validity error : Error validating datatype string
monuments.gpx:1010: element cmt: Relax-NG validity error : Element cmt failed to validate content
monuments.gpx:2054: element cmt: Relax-NG validity error : Error validating datatype string
monuments.gpx:2054: element cmt: Relax-NG validity error : Element cmt failed to validate content
monuments.gpx fails to validate
```

**Comment by mro** [comment]
some issues are fixed in 55011f4, while some remain, see https://travis-ci.org/Megalithicirelandmap/Megalithicirelandmap.github.io#L139

The remaining ones are due to uppercase letters in the anchor part which weren't used in th past.

Do you want to use mixed-case in the future (I'll make the check less scrupulous in this case) or stick with lowercase (you just change the spelling in anchor and reference)?

Edit: typo 'used' -> 'use'

**Comment by Megalithicireland** [comment]
Hi Marcus
I see the 4 corrections you made. A space needs %20 inserting?
We will leave as is I will do the corrections if I can find them
Thanks

**Comment by mro** [comment]
fixed with 0fa49df
